src/pages/docs/feature-toggles/index.md

```diff
@@ -0,0 +1,191 @@
+:::
+
+For applications deployed by Octopus, the recommended way is to have Octopus inject the client identifier as part of deployment, for example by injecting it into a configuration file or environment variable. The client identifier is made available via the Octopus variable `Octopus.FeatureToggles.ClientIdentifier`.   
+
+For applications not deployed by Octopus, or cannot have the client identifier supplied during deployment for any reason, the client identifier can be obtained via the portal UI, as shown below.
+
+![Client identifier preview menu item](/docs/feature-toggles/client-identifier-preview-menu-item.png "width=500")
+
+![Client identifier preview UI](/docs/feature-toggles/client-identifier-preview.png "width=500")
+
+The previewed client identifier may then be copied into your application configuration.
+
+For example, an ASP.NET application could have an `appsettings.json` file which contained the following: 
+
+```json
+{
+  "FeatureToggles": {
+    "ClientId": "#{Octopus.FeatureToggles.ClientIdentifier}"
+  }
+}
+```
+
+This would be transformed during deployment by Octopus to contain the correct client identifier for the current Project and Environment.
+
+This would then be used during application startup to configure the OpenFeature with the Octopus Provider, similar to:
+
+```cs
+// Retrieve client identifier from config 
+var builder = WebApplication.CreateBuilder(args);
+var octopusFeatureTogglesClientId = builder.Configuration["FeatureToggles:ClientId"] ?? "";
+
+// Instantiate the Octopus Provider
+var octopusProvider = new OctopusFeatureProvider(new OctopusFeatureConfiguration(octopusFeatureTogglesClientId));
+
+// Set Octopus as the OpenFeature provider
+await OpenFeature.Api.Instance.SetProviderAsync(octopusProvider);
+```
+
+### Evaluate a Toggle
+
+The [Provider](#providers) for each language documents how to evaluate toggles.
+
+You will need the Toggle slug in order to reference the toggle in code. This can be found in the Octopus portal:
+
+![Feature Toggle slug](/docs/feature-toggles/feature-toggle-slug.png "width=500")
+
+Below is an example of evaluating the toggle with slug `dark-mode` in C#:
+
+```cs
+var darkModeEnabled = await featureClient.GetBooleanValueAsync("dark-mode", false);
+```
+
+The second argument is the default value. Read more about [default values](#default-values) below.
+
+### Rollout 
+To enable your toggle for an environment, add the environment to the Toggle. 
+
+![Add Environment button](/docs/feature-toggles/add-environment-button.png "width=500")
+
+Select your environment, and whether you want the toggle on or off.
+
+![Add Environment dialog](/docs/feature-toggles/add-environment-dialog.png "width=500")
+
+You can additionally target specific [Tenants](#tenants) or [User Segments](#segments).
+
+## Providers {#providers}
```

Comment:
Suggestion: put a callout to say if people want a provider that isn't included below to get in contact with us.